contrib/lang/clojure/packages.el

```diff
@@ -77,7 +77,7 @@ which require an initialization must be listed explicitly in the list.")
         "mgv" 'cider-jump-to-var
         "mgr" 'cider-jump-to-resource
         "mge" 'cider-jump-to-compilation-error
-        "mgs" 'cider-jump
```

Comment:
`m g g` is the binding for go to thing under point, so go to definition is generally put under `m g g`